packages/core/src/render3/component.ts

```diff
@@ -17,94 +19,122 @@ import {notImplemented, stringify} from './util';
 
 
 
-/**
- * Options which control how the component should be bootstrapped.
- */
-export interface CreateComponentOptionArgs {
-  /**
-   * Which renderer factory to use.
-   */
+/** Options that control how the component should be bootstrapped. */
+export interface CreateComponentOptions {
+  /** Which renderer to use. */
+  renderer?: Renderer3;
```

Comment:
I think that is a bad merge. There is no `renderer` any more just `rendereFactory`

Reply:
Fixed it for you.